packages/google_sign_in/google_sign_in/ios/Classes/FLTGoogleSignInPlugin_Private.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Unfortunately I had to make this a public header to make it accessible to the RunnerTest.

Reply:
Maybe I should rename to `FLTGoogleSignInPlugin_Test.h`

Reply:
Just renamed it.

Reply:
It would be kind of ugly, but couldn't the test do a path-based include of the private header instead of getting it via the module import?

Reply:
Alternately, you could do the thing that I almost always tell people *not* to do (in favor of making an internal header like you did) and re-declare the method you need at the top of the test file. I've never liked it, but it seems better than actually making the private header public in the module.

Reply:
How about a `Test` module so it needs to be explicitly imported?
```obj-c
@import google_sign_in;
@import google_sign_in.Test;
```